public/locales/en/home.json

```diff
@@ -27,6 +27,10 @@
     },
     "clear": "Clear"
   },
+  "links": {
+    "sendDocumentTitle": "Send Report Document with Web",
```

Comment:
we can use Send Ballot Report with Web and Phone instead of Document.

Reply:
okay, working on it!